### ISSUE TITLE
Tracking: choice of base core calculus

### ISSUE THREAD

**Issue by mrkgnao** [mrkgnao/shorshe#1]
Currently the plan is to start with Oxide and then bolt some sort of dependent typing and effect tracking on top of that, but I'm open to alternatives.

**Comment by xldenis** [comment]
By Oxide do you mean the system exposed in https://arxiv.org/pdf/1903.00982.pdf ?

**Comment by mrkgnao** [comment]
Correct. I'm working from a more up-to-date version privately shared with me by @aatxe, though.

**Comment by xldenis** [comment]
Ah, well there's not much I can contribute until you have time to share your conclusions.

**Comment by mrkgnao** [comment]
Haha, that's fair. Sharing links to any resources you know of that might be
relevant would be more than enough by way of contribution!
